src/components/AppMethodPrimary.vue

```diff
@@ -60,28 +86,32 @@ export default {
     return {
       selectedFilter: '',
       selectedFind: '',
-      options: [
-        'add items or other arrays',
-        'remove items',
-        'find items',
-        'walk over items',
-        'return a string',
-        'order an array',
-        'something else'
-      ]
     }
   },
   computed: {
-    ...mapState([
-      'adding',
+    ...mapLocalizedState([
+      'adding', //: state => localizedState(state, 'adding'),
       'removing',
       'iterate',
       'string',
       'ordering',
       'other',
       'find'
-    ])
+    ]),
+    options: function() {
+      return this.$t('primaryOptions');
+    }
   },
+    /*...mapState([
+      'adding',
+      'removing',
+      'iterate',
+      'string',
+      'ordering',
+      'other',
+      'find'
+    ].map(prop => `${store.getters.curLanguage}.${prop}`))*/
```

Comment:
And this, can this be deleted as well?

Reply:
Yes, this is not required. It wasn't working.